iOSClient/Networking/NCOperationQueue.swift

```diff
@@ -135,7 +135,7 @@ import NextcloudKit
             if metadata.iconName.isEmpty {
                 cell?.filePreviewImageView?.image = NCBrandColor.cacheImages.file
             } else {
-                cell?.filePreviewImageView?.image = UIImage(named: metadata.iconName)
+                cell?.filePreviewImageView?.image = metadata.fileExtension == "odg" ? UIImage(named: "diagram") : UIImage(named: metadata.iconName)
```

Comment:
Can ODG file handled in metadata?
This has to be usptreamed as NC doesn't have this change.